CMakeLists.txt

```diff
@@ -501,13 +504,33 @@ endif()
 # BUILTINS
 #----------------------------------------------------------------------------#
 
-include(CudaRdcUtils)
-
 if(CELERITAS_BUILTIN)
   include(FetchContent)
   add_subdirectory(external)
 endif()
 
+#----------------------------------------------------------------------------#
+# RDC SUPPORT FOR VECGEOM+CUDA
+#----------------------------------------------------------------------------#
+
+# Load RDC utils using CMAKE_MODULE_PATH
+include(CudaRdcUtils)
+
+# Load our local copy if it's newer
+if(NOT CUDA_RDC_VERSION)
+   set(_loaded_CUDA_RDC_VERSION 0)
+else()
+   set(_loaded_CUDA_RDC_VERSION ${CUDA_RDC_VERSION})
+endif()
+include("${_LOCAL_RDCUTILS_FILENAME}")
```

Comment:
Isn't that always empty/not-set ... at least at the first run?

Reply:
Assuming no one used that variable name, yeah, but I thought you've knocked me before on not explicitly clearing variables 😉